src/frontends/common/include/openvino/frontend/plugin_loader.hpp

```diff
@@ -15,6 +15,23 @@ static const char PathSeparator[] = ":";
 namespace ov {
 namespace frontend {
 
+/// \brief Internal data structure holding by each frontend. Includes library handle and extensions.
+class FrontEndSharedData {
```

Comment:
Let's try to avoid having this header in 'public API'. If plugins need to access to extensions, maybe 'FrontEnd' class should have a kind of 'get_extensions' method

Reply:
The initial idea was to avoid new structures and headers in the public API.

Reply:
reverted